packages/inter-protocol/src/price/roundsManager.js

```diff
@@ -435,8 +435,10 @@ export const prepareRoundsManagerKit = baggage =>
             );
           }
 
-          if (status.lastReportedRound >= roundId)
+          if (status.lastReportedRound >= roundId) {
             return 'cannot report on previous rounds';
+          }
+
```

Comment:
Seems like an unnecessary change for the release branch

Reply:
dropped.

Reply:
removed.